apps/01PigLatin.js

```diff
@@ -7,8 +7,43 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
```

Comment:
Hi Rebecca! I think you also have to answer some of the concept questions. They can be found here:

https://gist.github.com/paulrenenichols/099c38009a569d603a098060f6cc3018